examples/next/with-emailpassword/config/frontendConfig.tsx

```diff
@@ -8,7 +8,12 @@ export let frontendConfig = () => {
         appInfo,
         // recipeList contains all the modules that you want to
         // use from SuperTokens. See the full list here: https://supertokens.com/docs/guides
-        recipeList: [EmailPasswordReact.init(), SessionReact.init()],
+        recipeList: [
+            EmailPasswordReact.init({
+                useShadowDom: false,
+            }),
```

Comment:
why is this needed?

Reply:
I've removed it, but the app was failing to render in "dev mode".